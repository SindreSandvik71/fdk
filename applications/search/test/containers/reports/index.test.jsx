import React from 'react';
import { shallow } from 'enzyme';
import ReportsPage from '../../../src/containers/reports';

test('should render ReportsPage correctly', () => {
  const wrapper = shallow(<ReportsPage />);
  expect(wrapper).toMatchSnapshot();
});
