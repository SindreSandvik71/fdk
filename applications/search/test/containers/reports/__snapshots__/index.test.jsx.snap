// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ReportsPage correctly 1`] = `
<div>
  <ResultsDatasetsReport
    selectedLanguageCode={null}
  />
</div>
`;
